app/controllers/brexit_checker_controller.rb

```diff
@@ -34,6 +34,9 @@ def results
     all_actions = BrexitChecker::Action.load_all
     @criteria = BrexitChecker::Criterion.load_by(criteria_keys)
     @actions = filter_items(all_actions, criteria_keys)
+    @audience_actions = @actions.group_by(&:audience)
```

Comment:
```@audience_actions``` seems to only be used here and I think it could be a normal variable instead of an instance one

Reply:
could call this grouped_actions maybe? To indicate better what it does

Reply:
I'm using `grouped_actions` on line 16 of `results_audiences.rb`. I'd be concerned about conflating the two?

Generally audiences are the highest level of subgroup within actions, if we think it's a confusing name would it not be better to rename that everywhere?